samples/v1.1/Tests/ImageSet.Stretch.json

```diff
@@ -1,907 +1,1046 @@
 {
-    "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
-    "type": "AdaptiveCard",
-    "version": "1.0",
-    "body": [
-        {
-            "type": "TextBlock",
-            "text": "ImageSet: ImageSize: small"
-        },
-        {
-            "type": "ImageSet",
-            "imageSize": "small",
-            "images": [
-                {
-                    "type": "Image",
-                    "backgroundColor": "#FF0000FF",
-                    "url": "https://adaptivecards.io/content/cats/1.png"
```

Comment:
⭐**Pro Tip**⭐
View this file using the "hide whitespace changes" option in GitHub's reviewing tool. For some reason, the diff appears as the entire file even though the tabs were maintained (might be the line endings? idk).